### PR TITLE
[MIG] stock_removal_location_by_priority: Migration to 16.0

### PR DESCRIPTION
**Pull request by aritzolea**
Standard migration

### CODE REVIEW

File: stock_removal_location_by_priority/models/stock_quant.py [hunk 1]
```diff
@@ -19,9 +19,9 @@ def _get_removal_strategy_order(self, removal_strategy):
             "stock_removal_location_by_priority.group_removal_priority"
         ):
             if removal_strategy == "fifo":
-                return "in_date ASC NULLS FIRST, removal_priority ASC, id"
```
Comment (rousseldenis) [inline stock_removal_location_by_priority/models/stock_quant.py]: Why this change ?

Comment (aritzolea) [reply stock_removal_location_by_priority/models/stock_quant.py]: `in_date` field is required on 16.0 (it wasn't on 14.0) so I guessed it's no longer needed

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by xAdrianCif — APPROVED** [review]
tech review ok, thanks!

**Review by AungKoKoLin1997 — APPROVED** [review]
Code Review(LGTM)
